src/App.jsx

```diff
@@ -324,7 +345,11 @@ const MainContent = ({
         <Count>
           {(keywords || coverage) && (
             <>
-              Showing results{" "}
+              Showing{" "}
+              <strong>
+                {amountOfResults(results.length)} out of {totalElements}
```

Comment:
Thinking about it method is currently returning 0 - 20, 20 - 40 but it should be 0 - 20, 21 - 40, 41 - 60 as the first result on the next page will be number 21 not number 20